Mathlib/CategoryTheory/Sites/Equivalence.lean

```diff
@@ -84,14 +81,26 @@ variable [e.TransportsGrothendieckTopology J K]
 theorem eq_inducedTopology_of_transports : K = e.inverse.inducedTopology J :=
   TransportsGrothendieckTopology.eq_inducedTopology
 
-instance : IsContinuous e.functor J K := by
+instance : e.functor.IsDenseSubsite J K := by
   rw [e.eq_inducedTopology_of_transports J K]
-  exact IsCoverDense.isContinuous _ _ _ (e.coverPreserving J)
-
-instance : IsContinuous e.inverse K J := by
-  rw [eq_inducedTopology_of_transports J K e]
   infer_instance
 
+instance : e.symm.TransportsGrothendieckTopology K J where
+  eq_inducedTopology := by
+    rw [e.eq_inducedTopology_of_transports J K]
+    ext X S
+    change _ ↔ _ ∈ J.sieves _
+    simp only [symm_inverse]
+    constructor
+    · intro h
+      convert J.pullback_stable (e.unitInv.app X) h
+      exact Sieve.functorPushforward_equivalence_eq_pullback e S
+    · intro h
+      convert J.pullback_stable (e.unit.app X) h
+      exact (Sieve.pullback_functorPushforward_equivalence_eq e S).symm
```

Comment:
```suggestion
    change _ ↔ _ ∈ J.sieves _
    simp [Sieve.functorPushforward_equivalence_eq_pullback]
```
We're probably missing simp lemmas at the `change`.

Reply:
Now that I think about it more, this is exactly the instance `instance : e.functor.IsDenseSubsite J (e.inverse.inducedTopology J)` a few line prior right? Maybe we could make `TransportsGrothendieckTopology` an abbrev or ditch it completely.

Reply:
You're right, it was very easy to just remove this. Let's see if CI finds errors elsewhere, but I think I fixed them all